### DISCUSSION TITLE
Scrollable Frame MoveToTop method?

### DISCUSSION THREAD

**Discussion by DominicCulyer** [TomSchimansky/CustomTkinter#1324] [General]
Hi,

I am using the same scrollable frame across various functions (passing the frame through function parameters), and the function then populates the frame with widgets. However, if the frame on page 1 (function 1) scrolls down and I click a button to take me to page 2 (function 2), then the view remains scrolled down on the second page, despite the widgets being above the shown view.

Is there a way around this? I can't see a method that scrolls the frame back to the top, so maybe this needs to be added?

Thanks
Dom

**Comment by SerVP22** [comment]
Hi.
How to scroll your frame down or up without a mouse?

**Comment by ElectricCandlelight** [comment]
```Python
Scrollable_frame_obj._parent_canvas.yview_moveto(0)
```
Not at my computer the now to test it but it should work

**Comment by SerVP22** [comment]
Scrollable_frame_obj._parent_canvas.yview_moveto(0) - Don't work
Scrollable_frame_obj._parent_canvas.yview_scroll(200, "units") - Work.
Thanks a lot
